tsconfig.json

```diff
@@ -15,6 +15,7 @@
         "resolveJsonModule": true,
         "downlevelIteration": true,
         "baseUrl": "./",
+        "types": ["node", "css-font-loading-module"],
```

Comment:
this significantly speeds up the compilation, it was well noticeable with bundling - by default typescript includes all `@types`, even ones that you don't use, e.g. lots from `@babel`) https://www.typescriptlang.org/tsconfig#types